### ISSUE TITLE
Show frequent apps/items until you start typing

### ISSUE THREAD

**Issue by isle9** [pop-os/launcher#158]
I'd like to see all of my frequent apps in a list when I open the launcher, before I start typing something in it.

**Comment by mmstick** [comment]
This should be easy to implement since 4eef0caae5deb3d608d4670af6457288490f85eb was merged.

**Comment by canadaduane** [comment]
I just did a little test, and I think this is simply a config change:

```bash
$ cat /usr/lib/pop-launcher/plugins/desktop_entries/plugin.ron 
(
    name: "Desktop Entries",
    description: "Query applications by their .desktop entries",
    query: (persistent: true),
    bin: (path: "desktop-entries"),
    icon: Name("new-window-symbolic"),
    history: true,
)
```

The `query: (persistent: true)` line makes the most frequently used apps show up in the result list even prior to typing a query term.

**Comment by isle9** [comment]
Yap, works. Thanks a lot.
Will I need to change the config with updates?

**Comment by canadaduane** [comment]
> Will I need to change the config with updates?

Yes, unfortunately. An alternative is to copy BOTH the `plugin.ron` file and the `desktop_entries` binary to your personal folder, below. This will override any system updates (which would be good for the `plugin.ron` file as it pins the configuration, but bad for the plugin binary, since it means system updates will be "ignored"):

```
$ mkdir -p ~/.local/share/pop-launcher/plugins/desktop_entries
$ cp /usr/lib/pop-launcher/plugins/desktop_entries/* \
        ~/.local/share/pop-launcher/plugins/desktop_entries
```

(Then edit the `~/.local/share/pop-launcher/plugins/desktop_entries/plugin.ron` config file as mentioned earlier in this thread).

**Comment by isle9** [comment]
Hopefully the plugin isn't updated that often then.

I've changed the title to better match the feature request. Ideally, this would be an option in the settings somewhere.
